rootio/streamers.go

```diff
@@ -224,6 +303,33 @@ func (tsb *tstreamerBasicType) Class() string {
 	return "TStreamerBasicType"
 }
 
+func (tsb *tstreamerBasicType) MarshalROOT(w *WBuffer) (int, error) {
```

Comment:
Not completely sure that's the correct thing to do.
But the read/write buffer round-trip test will tell us.

Could you enable it for the steamers?
Thanks